### DISCUSSION TITLE
./configs/rec/rec_svtrnet_ch.yml 这个配置可以训练  ./configs/rec/PP-OCRv4/ch_PP-OCRv4_rec.yml 这个配置即使设置batch size =1也爆显存，怎么修改这个配置让其能训练？

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#12594] [Q&A]
#### 运行环境 / Runtime Environment
- OS:
- Paddle: 2.3
- PaddleOCR: 2.7.5

#### 附件 / Appendix

./configs/rec/rec_svtrnet_ch.yml 这个配置可以训练
ch_PP-OCRv4_rec_svtr_large4lan.yml 这个配置也可以训练

./configs/rec/PP-OCRv4/ch_PP-OCRv4_rec.yml 这个配置即使设置batch size =1也爆显存，怎么修改这个配置让其能训练？

**Comment by GreatV** [comment]
升级一下paddle版本、换一个更大显存的服务器

**Comment by nissansz** [comment]
哪个svtr的配置得到的模型准确率最高？
